src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -417,4 +417,18 @@
         </HeaderTemplate>
     </BitNav>
 </div>
-
+<div class="basic-list">
+    <h2>BitList</h2>
+    <h3>Basic List</h3>
+    <div style="height:500px;overflow:auto">
+        <BitBasicList Items="people" LoadingText="please wait data is loading..." Virtualize="true" ItemPerPage="20">
+            <RowTemplate Context="person">
+                <div style=" border: 2px #8a8886 solid; border-radius: 10px; padding: 5px 20px; margin: 10px;">
+                    <p>Id : <strong >@person.Id</strong></p>
+                    <p>Full Name :<strong>@person.FirstName @person.LastName</strong>  </p>
+                    <p>Job: <strong>@person.Job</strong></p>
+                </div>                    
+            </RowTemplate>
+        </BitBasicList>
+    </div>
```

Comment:
please provide some more examples in order to explain other features of the BitBasicList